EDITORS.md

```diff
@@ -7,3 +7,13 @@ Code, use the official [Ruby LSP extension](https://github.com/Shopify/vscode-ru
 new H2 header in this file containing the instructions. -->
 
 - [Emacs LSP Mode](https://emacs-lsp.github.io/lsp-mode/page/lsp-ruby-lsp/)
+- [Emacs Eglot](#Emacs-Eglot)
+
+## Emacs Eglot
+
+[Eglot](https://github.com/joaotavora/eglot) runs solargraph server by default. To set it up with ruby-lsp you need to put that in your init file:
+```el
+(with-eval-after-load 'eglot
+ (add-to-list 'eglot-server-programs '((ruby-mode ruby-ts-mode) "ruby-lsp")))
```

Comment:
With solargraph there's a variable to set: https://emacs-lsp.github.io/lsp-mode/page/lsp-solargraph/#lsp-solargraph-use-bundler

Eglot is more pragmatic and indeed it's just a matter of the right entry in `eglot-server-programs`. The right one would be: `(add-to-list 'eglot-server-programs '((ruby-mode ruby-ts-mode) "bundle" "exec" "ruby-lsp"))`.

This configuration works on one of my machines but fail on another one. But it might be something else. I'll double check it and update the PR appropriately.

Reply:
Thanks!